examples/fluids/navierstokes.c

```diff
@@ -882,14 +911,19 @@ int main(int argc, char **argv) {
   }
   ierr = DMViewFromOptions(dm, NULL, "-dm_view"); CHKERRQ(ierr);
 
+  // Setup DM
   ierr = DMLocalizeCoordinates(dm); CHKERRQ(ierr);
   ierr = DMSetFromOptions(dm); CHKERRQ(ierr);
   ierr = SetUpDM(dm, problem, degree, &bc, &ctxSetup); CHKERRQ(ierr);
+
+  // Print FEM space information
   if (!test) {
     ierr = PetscPrintf(PETSC_COMM_WORLD,
-                       "Degree of FEM Space: %D\n",
+                       "Degree of FEM space: %D\n",
```

Comment:
At some point it might be nice to make this output match the rest of the PETSc examples. (maybe this is that point?)

Reply:
Done. Please let me know if it looks good to you

Reply:
Done. Let me know if it looks good to you. Thank you